04/app.js

```diff
@@ -1,5 +1,55 @@
 console.log('DOM');
 
+
+const grand = document.querySelector('nav')
+if(grand){
+    const ul = document.createElement('ul');
+    
+    grand.appendChild(ul);
+
+    const par = document.querySelector('ul');
+
+    if(par) {
+        const li1 = document.createElement('li');
+        par.appendChild(li1);
+
+        const a1 = document.createElement('a');
+        a1.setAttribute('href','/');
+        a1.innerText = 'start';
+        li1.appendChild(a1);
+
+        
+    }
+    if(par) {
```

Comment:
Skoro taki element utworzyliśmy to nie ma sens sprawdzać czy istnieje (bo wiemy, że tak jest)